config/routes.rb

```diff
@@ -7,6 +7,10 @@
   # Static pages
   get "/privacy", to: "static_pages#privacy"
 
+  # High voltage configuration. Matches routes from the root of the domain. Uses
```

Comment:
Would this be better at the bottom of the stack so we don't evaluate every request for another page against this?